app/services/exports/export_trainees_to_file_service.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require "csv"
+
+# This service is used to export trainees to a CSV file.
+class Exports::ExportTraineesToFileService < ExportToFileServiceBase
```

Comment:
I don't see an usages for this. Something for later on?

Reply:
Potentially for later on, but this also is useful now for a full test of csv export as seen here:

https://github.com/DFE-Digital/register-trainee-teachers/blob/dc577eeaa238c981aa36007d26b55e8bde96b1dc/spec/services/exports/export_trainees_to_file_service_spec.rb